source/backend/support/task.cpp

```diff
@@ -103,7 +103,7 @@ POV_LONG Task::ConsumedCPUTime() const
 void Task::Start(const boost::function0<void>& completion)
 {
     if((done == false) && (taskThread == NULL))
-        taskThread = NewBoostThread(boost::bind(&Task::TaskThread, this, completion), 1024 * 1024 * 2); // TODO - make stack size definable
+        taskThread = NewBoostThread(boost::bind(&Task::TaskThread, this, completion), POV_THREAD_STACK_SIZE);
```

Comment:
We still have a bit of time before beta.3, as the AVX stuff still needs some work as well, so there's no reason for excessive haste.

If you find you can't improve things beyond the current point, please do _not_ close the pull request; I think it's better than the status quo, so I guess I'd like to pull it either way, and maybe amend it later if I find the time.

As a general rule, if I just use regular comments on a pull request, consider them mere suggestions for further improvement; If I should find something that I consider a no-go, I would explicitly flag it as "change required" (or some such; can't tell off the top of my head what GitHub's official term for that review mechanism is).